test/conformance/device/urDeviceGetGlobalTimestamps.cpp

```diff
@@ -30,6 +51,50 @@ TEST_F(urDeviceGetGlobalTimestampTest, SuccessNoTimers) {
     }
 }
 
+TEST_F(urDeviceGetGlobalTimestampTest, SuccessSynchronizedTime) {
+    for (auto device : devices) {
+        uint64_t deviceStartTime, deviceEndTime = 0;
+        uint64_t hostStartTime, hostEndTime = 0;
+        uint64_t hostOnlyStartTime, hostOnlyEndTime = 0;
+
+        ASSERT_SUCCESS(urDeviceGetGlobalTimestamps(device, &deviceStartTime,
+                                                   &hostStartTime));
+        ASSERT_SUCCESS(
+            urDeviceGetGlobalTimestamps(device, nullptr, &hostOnlyStartTime));
+        ASSERT_NE(deviceStartTime, 0);
+        ASSERT_NE(hostStartTime, 0);
+        ASSERT_NE(hostOnlyStartTime, 0);
+        ASSERT_GE(hostOnlyStartTime, hostStartTime);
+
+        // wait for timers to increment
+        std::this_thread::sleep_for(std::chrono::seconds(5));
```

Comment:
could we make this less? 1 second is already a while for any modern processor and 5 seconds is a perceptible wait for the human running the test

Reply:
I'm happy to reduce this, this was just the time used by the OpenCL-CTS.  What would be a sensible amount - maybe 500ms?

Reply:
> this was just the time used by the OpenCL-CTS

ah yes, the test suite that's notoriously long running 😀 

>  What would be a sensible amount - maybe 500ms?

what's the resolution of the timers? perhaps 10x that amount?

Reply:
I was going to suggest keeping it the same order of magnitude and going with 1s, still human perceptible but not by much

Reply:
> what's the resolution of the timers? perhaps 10x that amount?
You can query the resolution with `UR_DEVICE_INFO_PROFILING_TIMER_RESOLUTION` - this would make it scalable. I quite like that idea.